pkg/devspace/kubectl/client.go

```diff
@@ -212,6 +212,8 @@ func NewClientBySelect(allowPrivate bool, switchContext bool, kubeLoader kubecon
 	return nil, errors.New("We should not reach this point")
 }
 
+var second = time.Second
```

Comment:
Why introduce this variable?

Reply:
This variable is overwritten in tests with one millisecond to shorten its duration.